src/boards/include/boards/hellbender_0001.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+#ifndef _BOARDS_HB_0001_H
+#define _BOARDS_HB_0001_H
+
+// pico_cmake_set PICO_PLATFORM=rp2350
+
+// For board detection
+#define HELLBENDER_0001
+
+// Pin aliases
+#define HB_0001_BLACKBOX_FLASH_CS_PIN 0 // on qspi bus
+
+#define HB_0001_IMU_CS_PIN 1 // also defined as PICO_DEFAULT_SPI_CSN_PIN
+#define HB_0001_IMU_IRQ_PIN 22
+#define HB_0001_IMU_CLKOUT_PIN 23
+
+#define HB_0001_5V_EN_PIN 14
+#define HB_0001_9V_EN_PIN 15
+
+#define HB_0001_LED_BLUE_PIN 6 // also defined as PICO_DEFAULT_LED_PIN
+#define HB_0001_LED_GREEN_PIN 7
+#define HB_0001_BUZZER_PIN 5
+
+#define HB_0001_GNSS_UART_TX_PIN 8
+#define HB_0001_GNSS_UART_RX_PIN 9
+
+#define HB_0001_DVTX_UART_TX_PIN 12
+#define HB_0001_DVTX_UART_RX_PIN 13
+#define HB_0001_DVTX_SBUS_PIN 36
+
+#define HB_0001_ESC_TELEM_RX_PIN 37
+
+#define HB_0001_OFFBOARD_WS2812_PIN 38
+#define HB_0001_ONBOARD_WS2812_PIN 39 // also defined as PICO_DEFAULT_WS2812_PIN
+
+#define HB_0001_SPARE_UART_TX_PIN 34
+#define HB_0001_SPARE_UART_RX_PIN 35
+
+#define HB_0001_I2C_CON_SDA_PIN 10
+#define HB_0001_I2C_CON_SCL_PIN 11
+
+#define HB_0001_FTRX_TX_PIN 20
+#define HB_0001_FTRX_RX_PIN 20
```

Comment:
Is it correct for these to both be the same?

Reply:
Oops, you're right, of course. Thanks for the catch.